docs/source/el7toel8/envars.md

```diff
@@ -67,6 +67,19 @@ Necessary to use in case you use any envar with the LEAPP_DEVEL prefix
 for the Leapp tool.
 
 
+## LEAPP_SERVICE_HOST
+Overrides the host of the service that leapp connects to to fetch necessary data
+files in case they are missing. It is required to specify the used protocol
+(`http://` or `https://`). Defaults to `https://cert.cloud.redhat.com`.
```

Comment:
```suggestion
## LEAPP_SERVICE_HOST
Overrides the host of the service to which leapp connects to fetch necessary data
files in case they are missing. It is required to specify the used protocol
(`http://` or `https://`). Defaults to `https://cert.cloud.redhat.com`.
```